### ISSUE TITLE
Add ability to customize commit messages when editing guides

### ISSUE THREAD

**Issue by durden** [pluralsight/guides-cms#20]
Currently it's difficult for editors to properly share the reasoning behind their edit suggestions with the original author.  We could use commit messages for this.  The idea is to add a small box to the UI when editing a guide you did not originally start.  This would get passed along in the commit message via the Github API for the commit to the branch where the edits live.

**Comment by NabaSadiaSiddiqui** [comment]
I agree with @durden . I am editing one of the articles directly on the website. Currently, all edits are saved with a very generic commit message along the lines "Updates to article_name". Every time I hit 'save', I need to be able to provide a small description of what I changed, and why. This is specially useful for the original author who would want to see the reasoning behind why something was added, removed or modified.

**Comment by durden** [comment]
Another idea would be to try using a code review tool for this feedback between editors and authors.  I recently came across the [Omniref](https://www.omniref.com) tool, which is a new approach to code review for github repositories.  It's also free to try for open source repos.  I could enable it if any authors and editors are willing to give it a trial run and provide feedback.  @prtkgpt what do you think?

**Comment by durden** [comment]
Adding a commit message box to the create/edit guide process has some other benefits:
1. Provides useful information in the github repository
   - Currently all the commit messages are very generic so looking through the `git log` is not very useful.
2. Allows authors to provide a summary of their guide when first creating it.
   - This summary could be useful for our sharing tools because we could automatically include the summary for facebook/tweets/etc.
